docs/monitoring.md

```diff
@@ -0,0 +1,221 @@
+# Enabling monitoring {#ovms_docs_monitoring}
+
+## Introduction to Metrics
+
+This document describes enabling and usage of different metrics in OVMS server that can be used for:
+
+- Providing performance statistics for monitoring and benchmarking purposes
+
+- Ability to auto scale model server instances in Kubernetes based on application related metrics
+
+> **NOTE**: currently, Metrics feature is released as a preview feature.
+
+Internal metrics remove dependencies on the 3rd party components and allow tracking performance without extra communication hop.
+
+It also exposes metrics which are not related to the traffic. 
+
+For example number of Idle execution streams, which models were used, model load time etc, which device was used.
+
+There are four types of metrics: 
+
+Counter � counts the number of specific events 
+
+Gauge � shows current state of the ovms or model 
+
+Histograms � shows distributions of counters per range 
+
+Summary � shows quantiles � will not be used on the server side 
+
+For more information please visit [Prometheus metrics detailed description](https://prometheus.io/docs/concepts/metric_types/)
+
+## Enable metrics
+
+By default, the metrics feature is disabled during standard server usage.
+
+To enable default metrics set you need to specify the metrics_enabled flag or json setting:
+
+CLI
+
+   ```bash
+         docker run --rm -d -v ${PWD}/models/resnet-50-tf:/opt/model -p 9001:9001 -p 9002:9002 openvino/model_server:latest \
+               --model_path /opt/model --model_name resnet --port 9001 \
+               --rest_port 9002 \
+               --metrics_enabled
+   ```
+
+CONFIG CMD
+
+   ```bash
+         docker run --rm -d -v -d -v ${PWD}/workspace:/workspace openvino/model_server --config_path /workspace/config.json -p 9001:9001 -p 9002:9002 openvino/model_server:latest \
```

Comment:
use standard ports like with have in all demos

Reply:
Ok.

Reply:
in other samples there is used rest port 8000. can be make the same in the command example?

Reply:
Fixed in new version.